### DISCUSSION TITLE
What is the easy and right way to do CRUD in Prisma and Pothos.

### DISCUSSION THREAD

**Discussion by ccfiel** [hayes/pothos#1037] [Q&A] [answered]
What is the easy and proper way to do CRUD in Prisma and Pothos? Like what Pothos has done in prismaConnection it is easy to do pagination.

**Comment by hayes** [comment, marked as answer]
There's the prisma-utils plugin you could check out, and I think there are a number of third party libraries that can auto generate crud schemas from your Prisma schema if that's what you're looking for

**Comment by ccfiel** [comment]
Thanks 👍
